app/views/govuk_publishing_components/components/_chevron_banner.html.erb

```diff
@@ -0,0 +1,23 @@
+<%
+  href ||= false
+  text ||= false
+  hover_border ||= false
+
+  data_attributes ||= {}
+  data_attributes[:module] = 'track-click' if data_attributes.any?
+%>
+<% if href && text %>
+  <!-- This assumes the application calling the component supports passing :extra_headers into the layout. -->
+  <!-- If not, add the following to your application layout file: `<%= yield :extra_headers %>` -->
```

Comment:
This comment is appearing the DOM, perhaps turn it into a ruby comment?

<img width="534" alt="Screen Shot 2019-09-05 at 10 49 10" src="https://user-images.githubusercontent.com/861310/64331542-e2631b00-cfca-11e9-9c04-d41ec26aa390.png">

Reply:
Or maybe even in the component guide?